wiki/How_You_Can_Help!/tr.md

```diff
@@ -0,0 +1,75 @@
+  - [Mapping Tartışma](https://osu.ppy.sh/community/forums/56)
+  - [Storyboard oluşturma](https://osu.ppy.sh/community/forums/20)
+  - [Oynama & Sıralamalar](https://osu.ppy.sh/community/forums/13)
+- Sağ üstte yer alan linki kullanarak forumu okundu olarak işaretleyin.
+- Her bir yeni gönderiyi, okuyarak ve yapabildiğinizce yardım ederek, takip edin. Ayak uydurabildiğinizden emin olmak için okunmadı ikonlarını kullanabilirsiniz.
+- [Mevcut rehberleri](/wiki/Guides) okuyarak osu! bilginizi tazeleyebilirsiniz. Eğer oradaki rehberler sizi tatmin etmiyorsa, onlara katkıda bulunun.
+
+## Beatmap kontrol etmede/modlamada yardım edin!
+
+osu!, beatmaplerin dereceli olmaya ve herkes tarafından oynanmaya hazır olup olmadığını kontrol eden özel bir kontrol ekibine sahiptir. Ancak, bu sizin bu kategoriye katkıda bulunmanıza engel değildir, işte yardımcı olmak için yapabileceğiniz bazı şeyler:
+
+- [Beklemedeki Beatmapler](https://osu.ppy.sh/beatmapsets?s=pending) listesinde sevdiğiniz sanatçılara ya da şarkılara göz atın (veya rastgele bir tane seçin).
+- Beklemede ya da yapım aşamasında bulunan beatmapleri oynayın ve beatmapin tartışma sayfasına bir geri bildirim bırakın.
+- Teknik modlamanın temel prensiplerini ve nelerin beatmapleri dereceli yaptığını öğrenmek için [Modlama Rehberini](/wiki/Modding) ve [Derecelendirme Kriterlerini](/wiki/Ranking_Criteria) okuyun.
+- Oyun-içi sohbette bulunan `#modhelp` ve `#modreqs` kanallarına katılın ve muhabbete dahil olun (ya da sadece konuşmaları takip edin).
+- Eğer vaktiniz ve gayretiniz varsa, beatmap yaratıcılarının sizden mod istemesi için bir [Modlama Sırası](https://osu.ppy.sh/community/forums/60) oluşturun.
+
+## Kullanıcılar tarafından yürütülen topluluk projelerine katılın/yardım edin
+
+osu! teknik ve rekabetçi kişilerden oluşan büyüyen bir topluluğa sahiptir. Onlar meraklı tamircilerden, görmüş geçirmiş ritim oyunu gazilerine kadar çeşitlendirilebilir. Burada osu! için özel olarak yapılan pek çok önerge ve proje mevcut ve siz bunlara yardım edebilir, veya kendinizinki ile osu!'nun geleceğini belirleyebilirsiniz! 
+
+İşte katılmak için yapabileceğiniz birtakım şeyler:
+
+- [Turnuvalar](https://osu.ppy.sh/community/forums/55) forumuna göz atın. Turnuvalar epey zaman almakta ve aksamadan organize edilmesi oldukça fazla bir işgücü gerektirmekte, bazı durumlarda izleyici juri olarak yardımcı olabilirsiniz.
+- [Geliştirme](https://osu.ppy.sh/community/forums/2)'de neler olduğunu kontrol edin. Bu düşük aktivite forumu en ciddi projelerin bulunduğu yerdir.
+- Bir [beatmap projesine](https://osu.ppy.sh/community/forums/53) katılın ve diğerleriyle birlikte çalışmanın ne kadar eğlenceli olduğunu öğrenin!
+- Kendi projenizi başlatın! Dans eden pippi'yi geri getirmek olsun, veya sadece osu! için havalı bir şey yapmak olsun, her şey uygundur.
+
+## osu!'nun geleceğine katkıda bulunun
+
+2017 yılında başlayan osu!lazer projesi, bugün görmüş olduğunuz yeni web sayfası ile birlikte, osu!team'in mevcut odağıdır.
```

Comment:
```suggestion
2017 yılında başlayan osu!lazer projesi, bugün görmüş olduğunuz yeni web sayfası ile birlikte, osu! ekibinin mevcut odağıdır.
```

not a proper osu!name

Reply:
"osu!team" has a branding value, though - we've also left it like this in `The_Team`

Reply:
I know it can be a proper osu!name, but the original text doesn't use it as a proper name, but I'll leave it to your discretion.